packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -159,6 +184,44 @@ class DatePicker extends React.Component {
     }
   }
 
+  handleOnInputFocus(event) {
+    if (this.props.onInputFocus) {
+      this.props.onInputFocus(event);
+    }
+
+    if (!this.isDefaultDateAcceptable) {
+      this.handleChange(null, event);
+      this.isDefaultDateAcceptable = true;
+    }
+  }
+
+  handleOnCalendarButtonClick(event, onClick) {
+    if (this.props.onCalendarButtonClick) {
+      this.props.onCalendarButtonClick(event);
+    }
+
+    if (!this.isDefaultDateAcceptable && !this.validateDefaultDate()) {
+      this.handleChange(null, event);
+    } else if (onClick) {
+      onClick();
+      this.isDefaultDateAcceptable = true;
+    }
+  }
+
+  validateDefaultDate() {
+    let iSAccepatable = true;
```

Comment:
I'd keep the S lowercase (isAcceptable)

Reply:
Updated: https://github.com/cerner/terra-core/pull/974/commits/f120301502e3d390e72a669fce130604de572ae5